Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -453,9 +444,9 @@ GLubyte* SoftwareGLContext::gl_get_string(GLenum name)
 
     switch (name) {
     case GL_VENDOR:
-        return reinterpret_cast<GLubyte*>(const_cast<char*>(m_device_info.vendor_name.characters()));
+        return reinterpret_cast<GLubyte*>(const_cast<char*>(m_rasterizer.info.vendor_name.characters_without_null_termination()));
     case GL_RENDERER:
-        return reinterpret_cast<GLubyte*>(const_cast<char*>(m_device_info.device_name.characters()));
+        return reinterpret_cast<GLubyte*>(const_cast<char*>(m_rasterizer.info.device_name.characters_without_null_termination()));
```

Comment:
This violates the documentation for this function:
All strings are null-terminated.

https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glGetString.xhtml

Reply:
Same as above.

Reply:
Changed to be a `char const*` instead of a `StringView`